CanonicalLtd/landscape-client
# Import hirsute patches and fix tests

Import missing ubuntu patches and make testsuite pass on hirsute, including package handling regression.

Testing instructions:

For the regression,
* In a fresh hirsute vm/container
* edit `root-client.conf`
* `./scripts/landscape-client -c root-client.conf`
* `add-apt-repository ppa:simpoir/dummy-test-only`
* wait for packages to sync in the UI and install `dummy-test-broken`
* landscape should report the broken package [as a failure](https://imgur.com/6tR7ZZH)
